### PR TITLE
Feature/redesign repo card

### PR DESCRIPTION
**Pull request by gcor**

### CODE REVIEW

File: components/Slots/Slots.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import en from "javascript-time-ago/locale/en";
+import TimeAgo from "javascript-time-ago";
+
+TimeAgo.addDefaultLocale(en);
+
+const timeAgo = new TimeAgo("en-US");
+const DEFAULT_BRANCH_NAME = "main";
+
+export function Slots({ slots = [], url }) {
+  const slotsWithoutBot = slots.filter((s) => s.commitAuthor !== "web-flow");
```
Comment (sashachabin) [inline components/Slots/Slots.jsx:11]: Let's  add const:
`const GITHUB_BOT_AUTHOR = "web-flow"`

File: components/Slots/Slots.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+          commitAuthorAvatar,
+        }) => (
+          <li className={styles.slot} key={commitUrl}>
+            <a
+              href={branch === DEFAULT_BRANCH_NAME ? url : slotUrl}
+              target="_blank"
+              rel="noreferrer"
+              className={styles.slot__link}
+            >
+              {branch}, {timeAgo.format(new Date(date))}
```
Comment (sashachabin) [inline components/Slots/Slots.jsx:32]: Let's separate branch and time. 

By the way, time will be very good to put in `<time datetime="ISO-raw-date-format"></time>`.

We will have a hint on hover with real-time (absolute instead relative)

Comment (sashachabin) [reply components/Slots/Slots.jsx:32]: image

\+ And we will have fewer underline, which we seem to have overdone

File: components/Slots/Slots.jsx (hunk 1 above)
Comment (gcor) [reply components/Slots/Slots.jsx:11]: did it in this pr https://github.com/ekaterinburgdev/projects/pull/10
moved to the be

File: components/Slots/Slots.jsx (hunk 2 above)
Comment (gcor) [reply components/Slots/Slots.jsx:32]: move to time at in the https://github.com/ekaterinburgdev/projects/pull/10

**Pull request merged**